components/statistics/count.lua

```diff
@@ -242,7 +242,12 @@ function Count._baseConditions(args, isTournament)
 	local conditions = ConditionTree(BooleanOperator.all)
 
 	if args.game then
-		conditions:add{ConditionNode(ColumnName('game'), Comparator.eq, args.game)}
+		local gameIdentifier = Game.toIdentifier{game = args.game, useDefault = false}
+		local gameConditions = ConditionTree(BooleanOperator.any):add{
+			ConditionNode(ColumnName('game'), Comparator.eq, args.game),
+			gameIdentifier and ConditionNode(ColumnName('game'), Comparator.eq, gameIdentifier) or nil,
+		}
+		conditions:add(gameConditions)
```

Comment:
Wouldn't it be enough to query on the gameId, and not the input?

Reply:
are we sure we only store identifiers and not names anymwhere?
if yes sure i will adjust it